plugins/editor/key-extension.js

```diff
@@ -1,57 +1,37 @@
 'use strict';
 
-var CodeMirror = require('codemirror');
+var { findNext, findPrevious, moveByScrollUpLine,
+      moveByScrollDownLine, tab, print } = require('../../src/actions/keys');
 
 const keyExtension = {
-  setup: function(app, cm) {
+  setup: function(app) {
 
     const cmCommands = {
       findNext: {
         code: 'F3',
-        exec: function(evt) {
-          evt.preventDefault();
-          cm.execCommand('findNext');
-        }
+        exec: findNext
```

Comment:
events should be handled in function wrappers here and not passed to the actions